features/support/fixtures.rb

```diff
@@ -0,0 +1,163 @@
+module Fixtures
+  DIGITAL_SPECIALISTS_SERVICE = {
+    id: 'TO BE REPLACED',
```

Comment:
I'd set these to `nil` with a comment. Otherwise if one of them is a valid value it could accidentally be accepted by the API if we forget to replace it.